pkg/conversion/mapi2capi/aws_test.go

```diff
@@ -54,6 +57,13 @@ type awsMAPI2CAPIConversionInput struct {
 	expectedWarnings []string
 }
 
+type awsMAPI2CAPIMachinesetConversionInput struct {
+	machineSetBuilder machinebuilder.MachineSetBuilder
+	infra             *configv1.Infrastructure
+	expectedErrors    []string
+	expectedWarnings  []string
+}
+
```

Comment:
Perhaps wrap the DescribeTable in an outer Describe and keep all the test bits within that? Else the scope leaks into the main package right?

Reply:
Yes good point